docs/README.md

```diff
@@ -68,17 +59,16 @@
     1. Expand `Runners` section
     1. Add specific runners or use shared runners
 1. Add environment variables to the project
-    * Key `TOWER_HOST` with `https://your.tower.domain.com/` value
-    * Key `TOWER_USERNAME` with `towerbot` value
-    * Key `TOWER_PASSWORD` with `abcd1234` value
-    * Key `TOWER_CERTIFICATE` if you are using custom CA or self-signed
+    * `TOWER_HOST`, `TOWER_USERNAME` and `TOWER_PASSWORD`
+    * `TOWER_CERTIFICATE` if you are using custom CA or self-signed
       certificates, value should be path to certificate or you can use
       [variable of type
       File](https://docs.gitlab.com/ee/ci/variables/#custom-environment-variables-of-type-file)
 1. Add assets (YAML files) to the repository (folder, eg. `data/assets/`)
-    * You can download the already existing ones with `podman run --rm -v $PWD:/workdir:Z europ/atacac backup <LABEL_ID> /workdir/data/assets`
-1. Configure the validation scheme stored in `data/schemas/`
-    * You can check asset validating schema locally via `podman run --rm -v $PWD:/workdir:Z -e ASSETS_GLOB="/workdir/data/assets/*.yml" -e ASSETS_SCHEMA=/workdir/<schema-path> europ/atacac validate`
+    * You can download the already existing ones with `atacac backup <LABEL_ID> 'data/assets/*.yml'`
```

Comment:
This command has its prerequisites. You should add a reference to the section that describes how to run this *atacac*-command via container or installed *atacac* pip package.

Reply:
`pip3 install --user atacac` in "Prerequisites" section is not enough?

Reply:
It should be enough. Hopefully, it is not necessary to mention it here.